src/exec/use_pty/parent.rs

```diff
@@ -186,15 +185,15 @@ pub(crate) fn exec_pty(
     // FIXME (ogsudo): Retry if `/dev/tty` is revoked.
 
     // Flush the terminal
-    closure.pty_pipe.flush(&mut closure.user_tty).ok();
-    closure.user_tty.flush().ok();
+    closure.tty_pipe.flush_left().ok();
```

Comment:
could `flush_left` (and `flush_right`) call `flush` on the `left` (`right`) `Write` object in its implementation so that one does not need to manually call `flush` here?

Reply:
yeah!